LEAF_Request_Portal/sources/System.php

```diff
@@ -871,7 +871,7 @@ public function getPrimaryAdmin()
     public function setPrimaryAdmin()
     {
         $vars = array(':userID' => XSSHelpers::xscrub($_POST['userID']));
-        $resultArray = array('success' => false, 'response' => $res);
+        //$resultArray = array('success' => false, 'response' => $res);
```

Comment:
Can you remove this line if it's not being used?

Reply:
Sure - done.